src/dbwrapper.h

```diff
@@ -236,6 +237,7 @@ class CDBWrapper
                 .in_memory = fMemory,
                 .wipe_existing = fWipe,
                 .obfuscate_data = obfuscate,
+                .do_compact = gArgs.GetBoolArg("-forcecompactdb", false),
```

Comment:
In commit "CDBWrapper: Pass in -forcecompactdb via ::Options" (d29f09adfaed0abeff0243509cdd49e4f52ebf9a)

gArgs access doesn't really belong in dbwrapper.h header either